app/src/main/java/com/kodeco/android/coordplot/components/Map.kt

```diff
@@ -0,0 +1,38 @@
+package com.kodeco.android.coordplot.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.paint
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.R
+import com.kodeco.android.coordplot.ui.theme.AppTheme
+import com.kodeco.android.coordplot.ui.theme.primaryContainerDark
+
+@Composable
+fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
+    Box(
+        modifier = Modifier
+            .size(300.dp)
```

Comment:
You can also move these `dp` values and other constants up and out of the function to `private const val`s or `private val`s.

Reply:
hey @zlippard thanks for your feedback!